### PR TITLE
[SDK-4952]Updated Readme

### PR DESCRIPTION
**Pull request by GMSyrimis**
### What does this Pull Request do?
Updates the readme to include all projects (RecyclerView has a PR ready)

### Why is this Pull Request needed?
The old instructions are outdated

### Are there any points in the code the reviewer needs to double check?
No

### Are there any Pull Requests open in other repos which need to be merged with this?
Approving this would be helpful:
https://github.com/jwplayer/jwplayer-android-best-practice-apps/pull/28

#### Addresses Issue(s):

SDK-4952

### CODE REVIEW

**Review by henryhlee — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-See the README in the subdirectory for specific usage instructions.
+Refer to the README in each subdirectory for specific usage instructions.
 
 ## Sample Apps
 
-Currently this repository contains two sample apps:
+- **ChromecastDemo** - A simple implementation of our Chromecast module. Allows you to cast to any available Chromecast devices
+- **ChromecastFragmentDemo** - Similar to the ChromecastDemo app, this app uses the `JWPlayerSupportFragment` as the main video component
+- **DemoNativeControls** - A barebones implementation of a native UI utilizing Android `View` elements
+- **FullBackgroundAudio**	- A demo of how you can setup the player to allow for background audio playback
```
Comment (paulmandal) [inline README.md]: Some of these have a space after the demo name and others have a tab

**Review by paulmandal — CHANGES_REQUESTED** [review]
Left one comment re: spaces vs tabs, otherwise it looks good.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
-- Click on the Dependencies tab, then click on the green plus sign and choose Module dependency
-- Select 'jwplayer-android-sdk-x.y.z' and click OK
-- Open the AndroidManifest.xml file and replace {YOUR_LICENSE_KEY} with your license key
+In general the following usage instructions apply to most sample projects:
+
+- Clone this repository
+- Start up Android Studio choose **Open an Existing Android Studio Project**
+- Navigate to the location of the repository and select the root directory of the app you want to load.
+- Click **OK**
+- Lastly, open the **AndroidManifest.xml** file and replace `{YOUR_LICENSE_KEY` with your **Android SDK 3** license key. You can find the license key in your dashboard under **Players > Downloads & Keys**
```
Comment (paulmandal) [inline README.md]: Missing the closing brace on `YOUR_LICENSE_KEY`

**Review by paulmandal — APPROVED** [review]
LGTM!

**Pull request merged**